tests/models/test_models_unet_2d_condition.py

```diff
@@ -500,6 +527,214 @@ def test_model_xattn_padding(self):
                 keeplast_out
             ), "a mask with fewer tokens than condition, will be padded with 'keep' tokens. a 'discard-all' mask missing the final token is thus equivalent to a 'keep last' mask."
 
+    def test_lora_processors(self):
```

Comment:
Why add all those tests here?

Reply:
Oh good catch. I think something weird might have happened when I merged in upstream changes. I'll take a look here.